src/main/java/org/kiwiproject/dropwizard/util/admin/StandardAdminConfigurator.java

```diff
@@ -0,0 +1,117 @@
+import org.kiwiproject.dropwizard.util.config.KeystoreConfig;
+import org.kiwiproject.dropwizard.util.health.HttpConnectionsHealthCheck;
+import org.kiwiproject.dropwizard.util.health.keystore.ExpiringKeystoreHealthCheck;
+import org.kiwiproject.dropwizard.util.metrics.ServerLoadGauge;
+import org.kiwiproject.dropwizard.util.task.ServerLoadTask;
+
+import java.util.stream.Stream;
+
+/**
+ * Set of utilities to assist in configuring administrative functions of a Dropwizard service.
+ */
+public class StandardAdminConfigurator {
+
+    private final Environment environment;
+    private TlsContextConfiguration tlsConfiguration;
+
+    @Getter(AccessLevel.PACKAGE)
+    @VisibleForTesting
+    private final Double leasedWarningThreshold;
+
+    @Getter(AccessLevel.PACKAGE)
+    @VisibleForTesting
+    private final boolean registerExpiringKeystoreHealthCheck;
+
+    @Getter(AccessLevel.PACKAGE)
+    @VisibleForTesting
+    private final boolean registerConfigResource;
+
+    @Builder
+    private StandardAdminConfigurator(Environment environment,
+                                      Double leasedWarningThreshold,
+                                      TlsContextConfiguration tlsConfiguration,
+                                      Boolean registerExpiringKeystoreHealthCheck,
+                                      Boolean registerConfigResource) {
+
+        this.environment = requireNotNull(environment, "Environment is required");
+        this.leasedWarningThreshold = isNull(leasedWarningThreshold) ? HttpConnectionsHealthCheck.DEFAULT_WARNING_THRESHOLD : leasedWarningThreshold;
+        this.registerExpiringKeystoreHealthCheck = isNull(registerExpiringKeystoreHealthCheck) || registerExpiringKeystoreHealthCheck;
+        this.registerConfigResource = isNull(registerConfigResource) || registerConfigResource;
+
+        if (this.registerExpiringKeystoreHealthCheck) {
+            this.tlsConfiguration = requireNotNull(tlsConfiguration,
+                    "TlsConfiguration is required when registering ExpiringKeystoreHealthCheck");
+        }
+    }
+
+    /**
+     * Sets up a task to retrieve the server load for the machine the service is running on.
+     *
+     * @see ServerLoadTask
+     */
+    public void addTasks() {
+        environment.admin().addTask(new ServerLoadTask());
+    }
+
+    /**
+     * Adds the {@link ServerLoadGauge} metric to the {@link MetricRegistry} to track server load.
+     *
+     * @see ServerLoadGauge
+     */
+    public void addMetrics() {
+        var metrics = environment.metrics();
+        var metricName = MetricRegistry.name(ServerLoadGauge.NAME);
+
+        metrics.register(metricName, new ServerLoadGauge());
+    }
+
+    /**
+     * Adds various default Health Checks to the service.
+     * <ul>
+     *     <li>{@link HttpConnectionsHealthCheck}</li>
+     *     <li>{@link ExpiringKeystoreHealthCheck} (if configured to do so)</li>
+     * </ul>
+     */
+    public void addHealthChecks() {
```

Comment:
To make this class a little more usable in "pieces", should we have separate methods for each individual thing that we add as "standard", e.g. `addHttpConnectionsHealthCheck`, `add ExpiringKeystoreHealthCheck`, `addServerLoadGauge`, etc. and then rename the generic methods as `addStandardXxx` and again mention that "standard" is simply our definition of standard. This would permit using the individual pieces without being forced to use _all_ of them, _or_ usage by calling the `addStandardXxx` methods.

This means each of the things would be broken out, e.g. for tasks, metrics, health checks, etc.

If we do this, then renaming this class to just `AdminConfigurator` actually makes a little more sense, since you can add either individual things, or you can add our "standard" ones.